test/e2e/volume_provisioning.go

```diff
@@ -184,6 +186,70 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 			Expect(err).To(HaveOccurred())
 			framework.Logf(err.Error())
 		})
+
+		It("should test that deleting a claim before the volume is provisioned deletes the volume. [Volume]", func() {
```

Comment:
Ok, I was wondering because of creating 30 claims and their provisioned pvs, and then deleting them all with a wait for each delete...